feature_engine/creation/math_features.py

```diff
@@ -175,18 +240,16 @@ def __init__(
                         "The number of new feature names must coincide with the number "
                         "of functions."
                     )
-            else:
-                if len(new_variables_names) != 1:
-                    raise ValueError(
-                        "The number of new feature names must coincide with the number "
-                        "of functions."
-                    )
+
+        if not (isinstance(ddof, int) or isinstance(ddof, float)):
+            raise ValueError("The type of ddof has to be Integer or Float")
```

Comment:
error message needs to contain "Got {ddof} instead." to help user undestand why the error